static-site/src/pages/404.jsx

```diff
@@ -5,24 +5,33 @@ import styled from "styled-components";
 import config from "../../data/SiteConfig";
 import NavBar from '../components/nav-bar';
 import Splash from "../components/splash";
+import UserDataWrapper from "../layouts/userDataWrapper";
 
 
 const FourOhFour = () => {
+  // Workaround so 404 page doesn't flash when pages are redirecting
```

Comment:
This is nice! ~It removes the flash for me when going straight to docs pages (with or without redirects) but the~ (this was a flash of the splash page, not the 404 page)

The 404 page still flashes when I click "login" or "logout". In particular, clicking logout causes 2-3 page flashes, including at least one 404. Can you replicate?

Reply:
After logging in and logging out, I no longer see 404 flashes.

Reply:
They're still there for me. This issue isn't blocking for this PR (the behavior's not introduced here) so i'm going to make an issue with GIFs so that you can see what i'm on about.